test/integration/project-page.test.js

```diff
@@ -177,3 +180,69 @@ describe('www-integration project-page signed in', () => {
         await expect(gfVisible).toBe(true);
     });
 });
+
+describe('www-integration project-creation signed in', () => {
```

Comment:
how do you decide whether to test signed it or signed out or both?

Reply:
I started with the tests being all signed out, and then added signed in tests as I went.  These last tests all involve creating and saving projects, so there needs to be a signed in user.  It's possible we could follow up with a signed out test for uploading a project, but that is already tested in the gui.  They could have different results, but it seems like that case would mostly be covered by the gui integration test and the signed in www test.